app/Providers/AppServiceProvider.php

```diff
@@ -23,6 +24,8 @@ public function register()
      */
     public function boot()
     {
-        //
+        Builder::macro('search', function ($field, $string) {
+            return $string ? $this->where($field, 'like', '%'.$string.'%') : $this;            
```

Comment:
I'm getting an error that "where" doesn't exist, even though my search is actually working. Weird.

Reply:
Same, I think this has to do with my epic version clashing going on in this project, not 100% sure yet.